storage/container_storage_interface/persistent-storage-csi-azure-stack-hub.adoc

```diff
@@ -0,0 +1,23 @@
+[id="persistent-storage-csi-azure-stack-hub"]
+= Azure Stack Hub CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure-stack-hub
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure Stack Hub Storage. Azure Stack Hub, which is part of the Azure Stack portfolio, allows you to run apps in an on-premises environment and deliver Azure services in your datacenter.
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Azure Stack Hub storage assets, {product-title} installs the Azure Stack Hub CSI Driver Operator and the Azure Stack Hub CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* The _Azure Stack Hub CSI Driver Operator_ provides a storage class (`managed-csi`), with "Standard_LRS" as the default storage account type, that you can use to create persistent volume claims (PVCs). The Azure Stack Hub CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
+
+* The _Azure Stack Hub CSI driver_ enables you to create and mount Azure Stack Hub PVs.
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Should this be _== Additional resources_ to make it obvious that the resources are for Azure Stack Hub CSI Driver Operator and not About CSI? The heading is tucked under About CSI, so it seems to be offering more information on CSI.

Reply:
Yes! Thanks!